src/halium-lxc-exec.sh

```diff
@@ -23,4 +23,4 @@ found_path=$(whereis -b -B ${ANDROID_SEARCH_PATH} -f ${TARGET_BINARY} | head -n
 unset LD_PRELOAD
 
 # Finally execute
-exec /usr/bin/lxc-attach -n ${LXC_CONTAINER_NAME} -- ${found_path/${LXC_CONTAINER_PATH}/} ${@}
+exec /usr/bin/lxc-attach -n ${LXC_CONTAINER_NAME} -- ${found_path/${LXC_CONTAINER_PATH}/} ${@} \;
```

Comment:
```
droidian@Xperia5:~$ sudo lxc-attach -n android echo test
test
droidian@Xperia5:~$ 
```

perhaps something specific to Halium-10? Either way, change works for me as well

Reply:
```
droidian@droidian:~$ sudo lxc-attach -n android echo test
[sudo] password for droidian: 

droidian@droidian:~$ 
```
Looks like a Halium-10 issue.